### PR TITLE
Lock openTSDB and Graphite listener address

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

File: opentsdb/opentsdb.go [hunk 1]
```diff
@@ -39,6 +39,7 @@ type Server struct {
 	wg       sync.WaitGroup
 
 	addr net.Addr
+	mu   sync.Mutex
```
Comment (otoolep) [inline opentsdb/opentsdb.go]: Just went with a simple mutex, since this code is not accessed very often.

**Comment by otoolep** [comment]
Fixes issues flagged by Go race detector.

**Comment by otoolep** [comment]
@corylanou

**Comment by corylanou** [comment]
It's strange that this ended up as a data race.  We initialize the servers outside of go routines, and then only ever read (not write) to that value.  I'm going to have to look through the code to figure out why this is getting reported as a data race.  In theory, even if it's really a "data race", it should never be possible to ever get dirty data from those values.

I'm not overly concerned, as the little overhead this creates is only on startup anyway.

+1

**Pull request merged**